### ISSUE TITLE
Web - Search - Missing ALT Text

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Comply-Chain#76]
The SVG left-pointing arrow is missing a title, description, or some piece of ARIA as part of the Home Page link. There is no ALT text or some kind of description for the SVG image of the magnifying glass that is decorative in the search box. The magnifying glass is displaying under the search INPUT filed in IE 11. They should at least have alt="" if nothing else. There are four other images most likely in the footer that I couldn’t see without having my tools in IE to find that are empty or missing ALT text.

There might be missing ALT text on other pages that I didn't have time to check so please those too.

Search - https://dolcontentdev.opadev.dol.gov/ilab/complychain/build/search

**Comment by KyleLivingstonDOL** [comment]
ZTI can we address the magnifying glass issue?

**Comment by Abychalil** [comment]
Alt text defined for the magnifying glass inside the search bar. Fixed

**Comment by jfc3-dol** [comment]
@Abychalil,

Thanks for adding the necessary ALT text. Will look at it when the next version/build is sent, and I have time to test.

**Comment by jfc3-dol** [comment]
There is now ALT text on the left pointing arrow, so I'm going to close the issue.
